common/rpc/interceptor/telemetry.go

```diff
@@ -230,6 +229,13 @@ func (ti *TelemetryInterceptor) emitActionMetric(
 				}
 			}
 		}
+		if hasMarker {
+			// Emit separate action metric for batch of markers.
+			// One workflow task response may contain multiple marker commands. Each marker will emit one
+			// command_RecordMarker_Xxx action metric. Depending on pricing model, you may want to ignore all individual
+			// command_RecordMarker_Xxx and use command_BatchMarkers instead.
+			metricsHandler.Counter(metrics.ActionCounter.GetMetricName()).Record(1, metrics.ActionType("command_BatchMarkers"))
```

Comment:
so `action` metric will double count? and people need to change the query to exclude either all of command_RecordMarker_Xxx or command_BatchMarkers to get the right count?

Reply:
Yes. Current OSS action metrics include all commands, but not all commands are billable actions.

Reply:
cc @feihuang after ^ is deployed, the sum of `action` metric across `action_type` will double count.